### PR TITLE
Add text changes for Log Search in Timeboards

### PR DESCRIPTION
**Pull request by MirandaKapin**
### What does this PR do?

Adds documentation for Log Search in Timeboards

### Motivation

New feature launch for Log Search in Timeboards

### Preview link

https://docs-staging.datadoghq.com/add-log-search-in-dashboards/graphing/dashboards/timeboard/#event-correlation-at-view-time

### Additional Notes

* Updated the Event Overlay as well
* Added new images
* Please review the text

### CODE REVIEW

File: content/en/graphing/dashboards/timeboard.md [hunk 1]
```diff
@@ -20,21 +20,31 @@ further_reading:
...
 Set up event correlation at design time by editing any graph on both Timeboards and Screenboards and adding events to the graph. You can find details about adding events [using the UI][1] or via the JSON interface further down the page.
 
-## Event Correlation at View Time
+### On whole dashboard
 
-{{< img src="graphing/dashboards/guides-eventcorrelation-searchbox.png" alt="guides event correlation" responsive="true" style="width:90%;">}}
+{{< img src="graphing/dashboards/open-search-event.png" alt="guides event correlation" responsive="true" style="width:75%;">}}
 
-Set up event correlation at view time by adding a query in the search box at the top left of any Timeboard dashboard window. This replaces any events added at design time, but applies the events to all graphs on that particular dashboard.
+Set up event correlation at view time by clicking on the "Search Events of Logs" link in the upper left hand side of the dashboard, and select "Events". Next, type a query in the search box at the top left of any Timeboard dashboard window. This replaces any events added at design time, but applies the events to all graphs on that particular dashboard as an overlay.
```
Comment (l0k0ms) [inline content/en/graphing/dashboards/timeboard.md]:
```suggestion
Set up event correlation at view time by clicking on the *Search Events of Logs* link in the upper left hand side of the dashboard, and select *Events*. Next, type a query in the search box at the top left of any Timeboard dashboard window. This replaces any events added at design time, but applies the events to all graphs on that particular dashboard as an overlay. 
```

File: content/en/graphing/dashboards/timeboard.md [hunk 2]
```diff
@@ -20,21 +20,31 @@ further_reading:
...
+{{< img src="graphing/dashboards/open-search-event.png" alt="guides event correlation" responsive="true" style="width:75%;">}}
 
-Set up event correlation at view time by adding a query in the search box at the top left of any Timeboard dashboard window. This replaces any events added at design time, but applies the events to all graphs on that particular dashboard.
+Set up event correlation at view time by clicking on the "Search Events of Logs" link in the upper left hand side of the dashboard, and select "Events". Next, type a query in the search box at the top left of any Timeboard dashboard window. This replaces any events added at design time, but applies the events to all graphs on that particular dashboard as an overlay. 
 
 ## Correlation between Logs and Metrics
 
+### Search for Logs in Timeboards
+
+Overlaying logs in your dashboard is easy. Click on the "Search Events of Logs" link in the upper left hand side of the dashboard, and select "Logs". This overlays the frequency of logs on your timeseries graphs. Click on the individual log line to view the full content.
```
Comment (l0k0ms) [inline content/en/graphing/dashboards/timeboard.md]:
```suggestion
Click on the *Search Events of Logs* link in the upper left hand side of the Timeboard, and select **Logs**. This overlays the frequency of logs on your timeseries widgets. Click on an individual log line to view its full content.
```

**Comment by MirandaKapin** [comment]
I don't feel strongly on the display of the page. But to me, it's tough when an image takes up most of the page which will happen if we want to display the results.

**Comment by MirandaKapin** [comment]
the events and logs have very different UI. If we do one, we should do the other

**Comment by ruthnaebeck** [comment]
@MirandaKapin - Looks like the log search results would need to be heavily redacted. We can just leave as it for now. Thanks!

**Pull request merged**